CHANGELOG.md

```diff
@@ -4,7 +4,16 @@ CHANGELOG
 Unreleased
 ----------
 
+Added:
 - GP-2098 Sorted input for bedtools
+
+Removed:
+- GP-2305 Remove downsampling and filtering from bam-qc-metrics
+
+v0.2.5: 2020-03-03
+------------------
+
+Added:
```

Comment:
I'm a bit confused about the ordering of these release notes; Is GP-2098 not being included in this release?

Reply:
No, it's not. GP-2098 addresses a failure with large BAM files being input to bedtools. But in our production workflow, bedtools output will be downsampled to 100,000 reads for the foreseeable future. So this is nice-to-have but not essential for this release.

Reply:
Per discussion on Slack, I've removed the "Unreleased" section of the changelog for greater clarity